_source/_posts/2020-01-21-angular-material-login.md

```diff
@@ -40,67 +41,56 @@ In this tutorial, I'll show you how to create a login form like the one above. Y
 To get started, you will need to install the Angular command-line tool. I will assume that you have some familiarity with JavaScript and that you have Node installed on your system together with the Node Package Manager `npm`. Open a terminal and type the following command.
 
 ```bash
-npm install -g @angular/cli@11
+npm install -g @angular/cli
 ```
 
-Depending on your operating system, you might have to run this using the `sudo` command. This will install the Angular command-line tool on your system. It lets you use the `ng` command to set up and manipulate Angular applications. To create a new application navigate to a directory of your choice and run the following.
+Depending on your operating system, you might have to run this using the `sudo` command. This will install the Angular command-line tool on your system. It lets you use the `ng` command to set up and manipulate Angular applications. In order to create an Angular app for the version we want, we'll use `npx` to create a new app. To create a new application navigate to a directory of your choice and run the following.
 
 ```bash
-ng new material-tic-tac-toe
+npx @angular/cli@13 new material-tic-tac-toe
```

Comment:
If you use this technique, please delete the former command that installs the Angular CLI globally.

Reply:
I'll update the former command for installing Angular CLI globally to add the qualification if it's not already globally available, but I think having a version of Angular CLI installed globally is helpful. Prepending npx for the rest of the Angular CLI commands becomes onerous and slows the commands down a lot.